docs/source/search-syntax.rst

```diff
@@ -80,21 +83,50 @@ For example:
 Run Attributes
 ~~~~~~~~~~~~~~
 
-The search syntax supports searching runs using two attributes: ``status`` and ``artifact_uri``. Both attributes have string values. Other fields in :py:class:`mlflow.entities.RunInfo` are :ref:`system_tags` that are searchable using the UI and the API. The search returns an error if you use other attribute names in the filter string. 
+You can search using two run attributes contained in :py:class:`mlflow.entities.RunInfo`: ``status`` and ``artifact_uri``. Both attributes have string values. 
+
+Other fields in ``mlflow.entities.RunInfo`` are :ref:`MLflow tags <system_tags>` that are searchable as ``tag`` identifiers (see :ref:`mlflow_tags` below); search returns an error if you use these names in the ``attributes`` identifier. 
 
 .. note::
   
   - The experiment ID is implicitly selected by the search API. 
   - A run's ``lifecycle_stage`` attribute is not allowed because it is already encoded as a part of the API's ``run_view_type`` field. To search for runs using ``run_id``, it is more efficient to use ``get_run`` APIs. 
   - The ``start_time`` and ``end_time`` attributes are not supported.
+  
+.. rubric:: Example
+
+.. code-block:: sql
+
+  attributes.artifact_uri
+
+
+.. _mlflow_tags:
+
+MLflow Tags
+~~~~~~~~~~~
+
+You can search for MLflow tags by enclosing the tag name in double quotes or backticks. For example, to search for the name of an MLflow run, specify ``tag."mlflow.runName"`` or ``tag.`mlflow.runName```. 
+
+.. note:: Search on Databricks does not support searching for a user (``tag."mlflow.user"``).
+
+.. rubric:: Examples
+
+.. code-block:: sql
+
+  tag."mlflow.runName"
+
+.. code-block:: sql
+
+  tag.`mlflow.parentRunId`
+
 
 Comparator
 ^^^^^^^^^^
 
 There are two classes of comparators: numeric and string.
 
 - Numeric comparators (``metrics``): ``=``, ``!=``, ``>``, ``>=``, ``<``, and ``<=``.
-- String comparators (``params``, ``tags``, and ``attributes``): ``=`` and ``!=``.
+- String comparators (``params``, ``tag``, and ``attributes``): ``=`` and ``!=``.
```

Comment:
let's be consistent and use either all plurals ``tags`` (or convert everything to singular)

Reply:
I didn't even realize MLflow parses both. I agree that we should keep it to plural everywhere and be consistent. I believe plural is also consistent with the names of the columns in the dataframe stuff I've been working on.

Reply:
@Zangr I realized that in the UI we have the columns as `Tags`, `Parameters` and `Metrics`, but if you put in bad syntax, the error message says : 
```Filter comparison is expected as either 'attribute.<key> <comparator> <value>', 'metric.<key> <comparator> <value>', 'tag.<key> <comparator> <value>', or 'param.<key> <comparator> <value>'.```
Could we change this to be plurals as well? Also, it's just slightly confusing that we name the column `Parameters` but you can't actually search by that. The placeholder text example does say `params.model...` though, which makes it more clear.

Reply:
@max-allen-db , I think this message is from BE. More specifically `search_utils._get_identifier`.